search.py

```diff
@@ -31,19 +35,60 @@ def run_search(dict_file, postings_file, queries_file, results_file):
     ############################################
     # Example code below to demonstrate how search.py may use InvertedIndex class to retrieve posting lists for term etc.....
 
-    inverted_index_class = InvertedIndex()
+    inverted_index_class = InvertedIndex(out_dict=dict_file, out_postings=postings_file)
+
+
+    f = open(queries_file, 'r')
+    for query in f.readlines():
+
+        query = QueryParser.parse(query)
```

Comment:
Possible to wrap in try except in case the query is not valid?
```
try:
            query = QueryParser.parse(query)
            posting_list = query.evaluate(inverted_index_class)
            print("Query: {}".format(query))
            print(" ".join(posting_list) + "\n")
except:
            print("Invalid Query format: "+query)``
```

Reply:
maybe we add this in later, cos for now its better if we have a bug it fails immediately